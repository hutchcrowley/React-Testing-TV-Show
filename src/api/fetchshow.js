import React, { useEffect } from "react";

import axios from "axios";

export const fetchShow = () => {
  const fetchShow = () => {
    return axios.get.then(res => {
      return res;
    });
  };
};
