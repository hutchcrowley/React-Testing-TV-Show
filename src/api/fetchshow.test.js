import { fetchShow } from "./fetchshow";

test("the data is returned from the API", () => {
  return fetchShow().then(res => {
      console.log(res.data)
    expect(res.data).toMatchSnapshot()
  });
});
