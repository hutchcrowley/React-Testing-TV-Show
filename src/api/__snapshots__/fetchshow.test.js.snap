// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the data is returned from the API 1`] = `
Object {
  "_embedded": Object {
    "episodes": Array [
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/553946",
          },
        },
        "airdate": "2016-07-15",
        "airstamp": "2016-07-15T12:00:00+00:00",
        "airtime": "",
        "id": 553946,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/67/168918.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/67/168918.jpg",
        },
        "name": "Chapter One: The Vanishing of Will Byers",
        "number": 1,
        "runtime": 60,
        "season": 1,
        "summary": "<p>A young boy mysteriously disappears, and his panicked mother demands that the police find him. Meanwhile, the boy's friends conduct their own search, and meet a mysterious girl in the forest.</p>",
        "url": "http://www.tvmaze.com/episodes/553946/stranger-things-1x01-chapter-one-the-vanishing-of-will-byers",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/578663",
          },
        },
        "airdate": "2016-07-15",
        "airstamp": "2016-07-15T12:00:00+00:00",
        "airtime": "",
        "id": 578663,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/72/181376.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/72/181376.jpg",
        },
        "name": "Chapter Two: The Weirdo on Maple Street",
        "number": 2,
        "runtime": 60,
        "season": 1,
        "summary": "<p>While the search for the missing Will continues, Joyce tells Jim about a call she apparently received from her son. Meanwhile, Jane warns Mike that there are bad people after her, and he realizes that she knows what happened to Will.</p>",
        "url": "http://www.tvmaze.com/episodes/578663/stranger-things-1x02-chapter-two-the-weirdo-on-maple-street",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/578664",
          },
        },
        "airdate": "2016-07-15",
        "airstamp": "2016-07-15T12:00:00+00:00",
        "airtime": "",
        "id": 578664,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/67/168920.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/67/168920.jpg",
        },
        "name": "Chapter Three: Holly, Jolly",
        "number": 3,
        "runtime": 60,
        "season": 1,
        "summary": "<p>While Nancy looks for a missing Barbara and realizes that Jonathan may have been the last person to see her, Mike and his friends go out with Jane to find the missing Will. Meanwhile, Jim tracks Will to the lab.</p>",
        "url": "http://www.tvmaze.com/episodes/578664/stranger-things-1x03-chapter-three-holly-jolly",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/578665",
          },
        },
        "airdate": "2016-07-15",
        "airstamp": "2016-07-15T12:00:00+00:00",
        "airtime": "",
        "id": 578665,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/67/168921.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/67/168921.jpg",
        },
        "name": "Chapter Four: The Body",
        "number": 4,
        "runtime": 60,
        "season": 1,
        "summary": "<p>Jim realizes that the government is covering something up about Will's death and begins a personal investigation. Meanwhile, Nancy discovers that Jonathan has information about Barbara's disappearance, while Mike and his friends smuggle Jane into the school so she can use the ham radio to contact Will.</p>",
        "url": "http://www.tvmaze.com/episodes/578665/stranger-things-1x04-chapter-four-the-body",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/578666",
          },
        },
        "airdate": "2016-07-15",
        "airstamp": "2016-07-15T12:00:00+00:00",
        "airtime": "",
        "id": 578666,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/67/168922.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/67/168922.jpg",
        },
        "name": "Chapter Five: The Flea and the Acrobat",
        "number": 5,
        "runtime": 60,
        "season": 1,
        "summary": "<p>Jim searches for Will at Hawkins Laboratory, but finds something unexpected. Meanwhile, Lonnie helps Joyce bury Will but reveals an ulterior motive for returning to town, while the boys find a way to locate Will but discover that Jane is opposing them.</p>",
        "url": "http://www.tvmaze.com/episodes/578666/stranger-things-1x05-chapter-five-the-flea-and-the-acrobat",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/578667",
          },
        },
        "airdate": "2016-07-15",
        "airstamp": "2016-07-15T12:00:00+00:00",
        "airtime": "",
        "id": 578667,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/67/168923.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/67/168923.jpg",
        },
        "name": "Chapter Six: The Monster",
        "number": 6,
        "runtime": 60,
        "season": 1,
        "summary": "<p>Jonathan manages to pull Nancy back to the real world from the Upside Down. Meanwhile, Lucas refuses to help with the search for Jane and goes to Hawkins Labs on his own, while Dustin and Mike look for the missing girl.</p>",
        "url": "http://www.tvmaze.com/episodes/578667/stranger-things-1x06-chapter-six-the-monster",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/578668",
          },
        },
        "airdate": "2016-07-15",
        "airstamp": "2016-07-15T12:00:00+00:00",
        "airtime": "",
        "id": 578668,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/67/168925.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/67/168925.jpg",
        },
        "name": "Chapter Seven: The Bathtub",
        "number": 7,
        "runtime": 60,
        "season": 1,
        "summary": "<p>Jim manages to bring everyone together so that they can join forces to reconnect Jane to the Upside Down and find Will. Meanwhile, the government closes in on Jane and the boys.</p>",
        "url": "http://www.tvmaze.com/episodes/578668/stranger-things-1x07-chapter-seven-the-bathtub",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/578669",
          },
        },
        "airdate": "2016-07-15",
        "airstamp": "2016-07-15T12:00:00+00:00",
        "airtime": "",
        "id": 578669,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/67/168926.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/67/168926.jpg",
        },
        "name": "Chapter Eight: The Upside Down",
        "number": 8,
        "runtime": 60,
        "season": 1,
        "summary": "<p>Jim convinces Brenner to let him and Joyce enter the Upside Down to find Will... unaware that Brenner plans to recover Jane no matter what it takes. Meanwhile, Nancy and Jonathan prepare to trap the monster at the Byers house, but receive an unexpected visitor.</p>",
        "url": "http://www.tvmaze.com/episodes/578669/stranger-things-1x08-chapter-eight-the-upside-down",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/909340",
          },
        },
        "airdate": "2017-10-27",
        "airstamp": "2017-10-27T12:00:00+00:00",
        "airtime": "",
        "id": 909340,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/132/331976.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/132/331976.jpg",
        },
        "name": "Chapter One: MADMAX",
        "number": 1,
        "runtime": 60,
        "season": 2,
        "summary": "<p>One year after the events with the Upside Down and the Demogorgon, Will meets with a government doctor. The boys discover that there's a new player in town, and Jim pays a visit to El.</p>",
        "url": "http://www.tvmaze.com/episodes/909340/stranger-things-2x01-chapter-one-madmax",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/909342",
          },
        },
        "airdate": "2017-10-27",
        "airstamp": "2017-10-27T12:00:00+00:00",
        "airtime": "",
        "id": 909342,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/132/332034.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/132/332034.jpg",
        },
        "name": "Chapter Two: Trick or Treat, Freak",
        "number": 2,
        "runtime": 60,
        "season": 2,
        "summary": "<p>The boys go trick-or-treating on Halloween, and Will has another vision. Meanwhile, El relieves the days following her escape from the Upside Down, and Dustin finds something in the garbage can.</p>",
        "url": "http://www.tvmaze.com/episodes/909342/stranger-things-2x02-chapter-two-trick-or-treat-freak",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/909343",
          },
        },
        "airdate": "2017-10-27",
        "airstamp": "2017-10-27T12:00:00+00:00",
        "airtime": "",
        "id": 909343,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/132/332039.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/132/332039.jpg",
        },
        "name": "Chapter Three: The Pollywog",
        "number": 3,
        "runtime": 60,
        "season": 2,
        "summary": "<p>Dustin takes in a stray and calls it D'Artagnan. However, his plans to show it to Mr. Clarke don't go as intended. Meanwhile, Max tries to convince Mike to let her join the group, El sneaks out to pay her friends a visit, and Will decides to take a stand and face his fears.</p>",
        "url": "http://www.tvmaze.com/episodes/909343/stranger-things-2x03-chapter-three-the-pollywog",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/909344",
          },
        },
        "airdate": "2017-10-27",
        "airstamp": "2017-10-27T12:00:00+00:00",
        "airtime": "",
        "id": 909344,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/132/332045.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/132/332045.jpg",
        },
        "name": "Chapter Four: Will the Wise",
        "number": 4,
        "runtime": 60,
        "season": 2,
        "summary": "<p>After his encounter with the shadow creature, Will's condition worsens. Meanwhile, Jim and El fight after Jim discovers that El disobeyed the rules, and Owens brings Nancy and Jonathan to Hawkins Labs for a chat.</p>",
        "url": "http://www.tvmaze.com/episodes/909344/stranger-things-2x04-chapter-four-will-the-wise",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/909345",
          },
        },
        "airdate": "2017-10-27",
        "airstamp": "2017-10-27T12:00:00+00:00",
        "airtime": "",
        "id": 909345,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/132/332050.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/132/332050.jpg",
        },
        "name": "Chapter Five: Dig Dug",
        "number": 5,
        "runtime": 60,
        "season": 2,
        "summary": "<p>Jim is trapped in the Upside Down, and Joyce enlists Bob to help find him. Meanwhile, Nancy and Jonathan go to Murray, and El learns about the circumstances surrounding her birth.</p>",
        "url": "http://www.tvmaze.com/episodes/909345/stranger-things-2x05-chapter-five-dig-dug",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/909346",
          },
        },
        "airdate": "2017-10-27",
        "airstamp": "2017-10-27T12:00:00+00:00",
        "airtime": "",
        "id": 909346,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/132/332052.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/132/332052.jpg",
        },
        "name": "Chapter Six: The Spy",
        "number": 6,
        "runtime": 60,
        "season": 2,
        "summary": "<p>Owens and his team try to find a way to end the shadow creature's influence on Will. Meanwhile, Dustin and Lucas find unlikely allies in their quest to find Dart.</p>",
        "url": "http://www.tvmaze.com/episodes/909346/stranger-things-2x06-chapter-six-the-spy",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/909347",
          },
        },
        "airdate": "2017-10-27",
        "airstamp": "2017-10-27T12:00:00+00:00",
        "airtime": "",
        "id": 909347,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/132/332055.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/132/332055.jpg",
        },
        "name": "Chapter Seven: The Lost Sister",
        "number": 7,
        "runtime": 60,
        "season": 2,
        "summary": "<p>El travels to Chicago to find her sister Eight, aka Kali, who has teamed up with other outcasts to kill the men who tormented her at Hawkins Labs.</p>",
        "url": "http://www.tvmaze.com/episodes/909347/stranger-things-2x07-chapter-seven-the-lost-sister",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/909348",
          },
        },
        "airdate": "2017-10-27",
        "airstamp": "2017-10-27T12:00:00+00:00",
        "airtime": "",
        "id": 909348,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/132/332059.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/132/332059.jpg",
        },
        "name": "Chapter Eight: The Mind Flayer",
        "number": 8,
        "runtime": 60,
        "season": 2,
        "summary": "<p>When the demo-dogs overrun Hawkins Labs, one of the group saves the others at the cost of their own life. When the others get back to the Byers house, they realize what they're facing and try to get answers from a captive Will.</p>",
        "url": "http://www.tvmaze.com/episodes/909348/stranger-things-2x08-chapter-eight-the-mind-flayer",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/909349",
          },
        },
        "airdate": "2017-10-27",
        "airstamp": "2017-10-27T12:00:00+00:00",
        "airtime": "",
        "id": 909349,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/132/332064.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/132/332064.jpg",
        },
        "name": "Chapter Nine: The Gate",
        "number": 9,
        "runtime": 60,
        "season": 2,
        "summary": "<p>The group splits up to close the gate, evict the Mind Flayer from Will, and drive the demo-dogs away from Hawkins Lab.</p>",
        "url": "http://www.tvmaze.com/episodes/909349/stranger-things-2x09-chapter-nine-the-gate",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/1576469",
          },
        },
        "airdate": "2019-07-04",
        "airstamp": "2019-07-04T12:00:00+00:00",
        "airtime": "",
        "id": 1576469,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/204/510098.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/204/510098.jpg",
        },
        "name": "Chapter One: Suzie, Do You Copy?",
        "number": 1,
        "runtime": 51,
        "season": 3,
        "summary": "<p>Things change over the summer: Jonathan, Nancy, Steve, and Billy get jobs; Dustin goes to science camp; El and Mike become an item; Lucas and Max almost become an item. Meanwhile, mysterious power outages plague Hawkins and rats start exploding.</p>",
        "url": "http://www.tvmaze.com/episodes/1576469/stranger-things-3x01-chapter-one-suzie-do-you-copy",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/1576470",
          },
        },
        "airdate": "2019-07-04",
        "airstamp": "2019-07-04T12:00:00+00:00",
        "airtime": "",
        "id": 1576470,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/204/511261.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/204/511261.jpg",
        },
        "name": "Chapter Two: The Mall Rats",
        "number": 2,
        "runtime": 50,
        "season": 3,
        "summary": "<p>While El and Max go shopping, Nancy and Wheeler follow up on the exploding rats. Billy takes a coworker on a field trip, and Joyce blows off dinner with Jim to meet with Mr. Clarke.</p>",
        "url": "http://www.tvmaze.com/episodes/1576470/stranger-things-3x02-chapter-two-the-mall-rats",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/1576471",
          },
        },
        "airdate": "2019-07-04",
        "airstamp": "2019-07-04T12:00:00+00:00",
        "airtime": "",
        "id": 1576471,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/204/511286.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/204/511286.jpg",
        },
        "name": "Chapter Three: The Case of the Missing Lifeguard",
        "number": 3,
        "runtime": 50,
        "season": 3,
        "summary": "<p>El goes astrally projecting and discovers that Billy has done something with Heather. While the girls try to find the missing lifeguard, Will tries to get Mike and Lucas interested in D&amp;D. Dustin and Steve spy on the mall shops, and Robin tries to decipher the Russian message.</p>",
        "url": "http://www.tvmaze.com/episodes/1576471/stranger-things-3x03-chapter-three-the-case-of-the-missing-lifeguard",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/1576472",
          },
        },
        "airdate": "2019-07-04",
        "airstamp": "2019-07-04T12:00:00+00:00",
        "airtime": "",
        "id": 1576472,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/204/511312.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/204/511312.jpg",
        },
        "name": "Chapter Four: The Sauna Test",
        "number": 4,
        "runtime": 53,
        "season": 3,
        "summary": "<p>Mike, Lucas, and Will recruit El and Max to help them learn who the Mind Flayer's host is. Robin, Steve, and Dustin recruit someone to sneak into Lynx, and Nancy and Jonathan are fired.</p>",
        "url": "http://www.tvmaze.com/episodes/1576472/stranger-things-3x04-chapter-four-the-sauna-test",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/1576473",
          },
        },
        "airdate": "2019-07-04",
        "airstamp": "2019-07-04T12:00:00+00:00",
        "airtime": "",
        "id": 1576473,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/204/510102.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/204/510102.jpg",
        },
        "name": "Chapter Five: The Flayed",
        "number": 5,
        "runtime": 52,
        "season": 3,
        "summary": "<p>The Scoop Group explore the tunnels beneath Starcourt, while Nancy and Jonathan recruit Mike and the others to help them discover what happen to the Holloways and find the Mind Flayer's base of operations.</p>",
        "url": "http://www.tvmaze.com/episodes/1576473/stranger-things-3x05-chapter-five-the-flayed",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/1576474",
          },
        },
        "airdate": "2019-07-04",
        "airstamp": "2019-07-04T12:00:00+00:00",
        "airtime": "",
        "id": 1576474,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/204/511681.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/204/511681.jpg",
        },
        "name": "Chapter Six: E Pluribus Unum",
        "number": 6,
        "runtime": 60,
        "season": 3,
        "summary": "<p>Dr. Alexei reveals what the Russians have been building, and Eleven sees where Billy has been. Dustin and Erica stage a daring rescue.</p>",
        "url": "http://www.tvmaze.com/episodes/1576474/stranger-things-3x06-chapter-six-e-pluribus-unum",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/1576475",
          },
        },
        "airdate": "2019-07-04",
        "airstamp": "2019-07-04T12:00:00+00:00",
        "airtime": "",
        "id": 1576475,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/204/511708.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/204/511708.jpg",
        },
        "name": "Chapter Seven: The Bite",
        "number": 7,
        "runtime": 55,
        "season": 3,
        "summary": "<p>El and the others defend themselves as the Flayer tracks them down. The Scoop Group escape the Russian base and hide in a movie theater, but the Russians close in on them.</p>",
        "url": "http://www.tvmaze.com/episodes/1576475/stranger-things-3x07-chapter-seven-the-bite",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/1576476",
          },
        },
        "airdate": "2019-07-04",
        "airstamp": "2019-07-04T12:00:00+00:00",
        "airtime": "",
        "id": 1576476,
        "image": Object {
          "medium": "http://static.tvmaze.com/uploads/images/medium_landscape/204/510105.jpg",
          "original": "http://static.tvmaze.com/uploads/images/original_untouched/204/510105.jpg",
        },
        "name": "Chapter Eight: The Battle of Starcourt",
        "number": 8,
        "runtime": 78,
        "season": 3,
        "summary": "<p>The Flayer comes to the mall to get El, and Dustin contacts Suzie to provide Jim and Joyce with needed information. El reminds Billy of his past, and Jim and Joyce have an argument about not arguing.</p>",
        "url": "http://www.tvmaze.com/episodes/1576476/stranger-things-3x08-chapter-eight-the-battle-of-starcourt",
      },
      Object {
        "_links": Object {
          "self": Object {
            "href": "http://api.tvmaze.com/episodes/1752754",
          },
        },
        "airdate": "",
        "airstamp": null,
        "airtime": "",
        "id": 1752754,
        "image": null,
        "name": "Chapter One: The Hellfire Club",
        "number": 1,
        "runtime": 60,
        "season": 4,
        "summary": null,
        "url": "http://www.tvmaze.com/episodes/1752754/stranger-things-4x01-chapter-one-the-hellfire-club",
      },
    ],
  },
  "_links": Object {
    "previousepisode": Object {
      "href": "http://api.tvmaze.com/episodes/1576476",
    },
    "self": Object {
      "href": "http://api.tvmaze.com/shows/2993",
    },
  },
  "externals": Object {
    "imdb": "tt4574334",
    "thetvdb": 305288,
    "tvrage": 48493,
  },
  "genres": Array [
    "Drama",
    "Fantasy",
    "Science-Fiction",
  ],
  "id": 2993,
  "image": Object {
    "medium": "http://static.tvmaze.com/uploads/images/medium_portrait/200/501942.jpg",
    "original": "http://static.tvmaze.com/uploads/images/original_untouched/200/501942.jpg",
  },
  "language": "English",
  "name": "Stranger Things",
  "network": null,
  "officialSite": "https://www.netflix.com/title/80057281",
  "premiered": "2016-07-15",
  "rating": Object {
    "average": 8.7,
  },
  "runtime": 60,
  "schedule": Object {
    "days": Array [
      "Thursday",
    ],
    "time": "",
  },
  "status": "Running",
  "summary": "<p>A love letter to the '80s classics that captivated a generation, <b>Stranger Things</b> is set in 1983 Indiana, where a young boy vanishes into thin air. As friends, family and local police search for answers, they are drawn into an extraordinary mystery involving top-secret government experiments, terrifying supernatural forces and one very strange little girl.</p>",
  "type": "Scripted",
  "updated": 1582785474,
  "url": "http://www.tvmaze.com/shows/2993/stranger-things",
  "webChannel": Object {
    "country": null,
    "id": 1,
    "name": "Netflix",
  },
  "weight": 98,
}
`;
